rpm/SPECS/cep.spec

```diff
@@ -164,6 +164,12 @@ rm -rf $RPM_BUILD_ROOT
 %{_install_dir}
 
 %changelog
+* Wed May 11 2022 Alvaro Vega <alvaro.vegagarcia@telefonica.com> 1.23.0
```

Comment:
CNR file should be flushed.

Reply:
done in https://github.com/telefonicaid/perseo-fe/pull/673/commits/dc3b459cfa71b1f71278c43c3f52a8761128352e